lib/taskana-core/src/test/java/acceptance/task/QueryTasksByWildcardSearchAccTest.java

```diff
@@ -61,6 +61,32 @@ void should_ReturnAllTasks_For_ProvidedSearchValueAndAdditionalParameters() {
     assertThat(foundTasks).hasSize(1);
   }
 
+  @WithAccessId(user = "admin")
+  @Test
+  void should_ReturnAllTasks_For_CaseInsensitiveValue() {
+    TaskService taskService = taskanaEngine.getTaskService();
+
+    WildcardSearchField[] wildcards = {WildcardSearchField.NAME};
+
+    List<TaskSummary> foundTasksSensitive =
+        taskService
+            .createTaskQuery()
+            .wildcardSearchFieldsIn(wildcards)
+            .wildcardSearchValueLike("%Wid%")
+            .list();
+
+    List<TaskSummary> foundTasksInSensitive =
```

Comment:
Maybe more meaningful to call it "foundTasksUpperCase" and "foundTasksLowerCase".

Reply:
Decided to go with foundTasksCaseInsensitive / foundTasksSensitive after consulting with Mustapha